Documentation/README.md

```diff
@@ -1,17 +1,26 @@
 # Hubble Documentation
 
+> The Hubble documentation has moved to [docs.cilium.io](https://docs.cilium.io/)
+
+This documentation is intended for users with **Cilium >=1.8**. If
+you are running Cilium 1.7, please follow [the documentation in the Hubble v0.5
+branch](https://github.com/cilium/hubble/tree/v0.5/Documentation).
+
 ## Getting Started
 
- * [Quickstart](quickstart.md)
- * [Concepts](concepts.md)
+ * [Quickstart](https://docs.cilium.io/en/latest/gettingstarted/hubble)
+ * [What is Hubble?](https://docs.cilium.io/en/latest/intro/#what-is-hubble)
+ * [Installation](https://docs.cilium.io/en/latest/gettingstarted/hubble-enable/)
```

Comment:
I'm not sure this page is intended to be linked to as is. It is indeed included in the relevant Cilium installation guides.

Reply:
I was a bit torn about this, since it might not be obvious to someone who is not reading carefully that Hubble is now part of Cilium. But I fixed the link to point to the Cilium installation section now.